source/transfer_list.rst

```diff
@@ -1034,7 +1072,43 @@ passed to later stages for intialisation of Mbed-TLS.
      - hdr_size + 0x8
      - Size of memory region.
 
+**DT formatted SPMC manifest entry layout (XFERLIST_DT_FFA_MANIFEST)**
```

Comment:
I've just noticed the "Mbed-TLS heap information" TE incorrectly starts a new section. This issue was already there but means this TE appears in the wrong section. Would be good if you can submit a fix for that too.

Reply:
I will move it back.
The intention was to retain order of TE ID.
It seems HEAP is 0x105